controller/rest/sigstore.go

```diff
@@ -191,31 +190,33 @@ func handlerSigstoreRootOfTrustGetAll(w http.ResponseWriter, r *http.Request, ps
 		return
 	}
 
-	rootsOfTrust, err := clusHelper.GetAllSigstoreRootsOfTrust()
+	clusRootsOfTrust, err := clusHelper.GetAllSigstoreRootsOfTrust()
 	if err != nil {
 		msg := fmt.Sprintf("could not retrieve sigstore roots of trust from kv store: %s", err.Error())
 		restRespErrorMessage(w, http.StatusInternalServerError, api.RESTErrFailReadCluster, msg)
 		return
 	}
 
-	resp := make(map[string]api.REST_SigstoreRootOfTrust_GET, len(rootsOfTrust))
-	for key, rootOfTrust := range rootsOfTrust {
-		restRootOfTrust := CLUSRootToRESTRoot_GET(rootOfTrust)
+	rootsOfTrust := []api.REST_SigstoreRootOfTrust_GET{}
+	for _, clusRootOfTrust := range clusRootsOfTrust {
+		rootOfTrust := CLUSRootToRESTRoot_GET(clusRootOfTrust)
 		if withVerifiers(r) {
-			verifiers, err := clusHelper.GetAllSigstoreVerifiersForRoot(key)
+			verifiers, err := clusHelper.GetAllSigstoreVerifiersForRoot(clusRootOfTrust.Name)
 			if err != nil {
-				msg := fmt.Sprintf("could not retrieve verifiers for root \"%s\": %s", key, err.Error())
+				msg := fmt.Sprintf("could not retrieve verifiers for root \"%s\": %s", clusRootOfTrust.Name, err.Error())
 				restRespErrorMessage(w, http.StatusInternalServerError, api.RESTErrFailReadCluster, msg)
 				return
 			}
-			restRootOfTrust.Verifiers = make(map[string]api.REST_SigstoreVerifier, len(verifiers))
-			for name, verifier := range verifiers {
-				restRootOfTrust.Verifiers[name] = CLUSVerifierToRESTVerifier(verifier)
+			for _, verifier := range verifiers {
+				rootOfTrust.Verifiers = append(rootOfTrust.Verifiers, CLUSVerifierToRESTVerifier(verifier))
 			}
 		}
-		resp[key] = restRootOfTrust
+		rootsOfTrust = append(rootsOfTrust, rootOfTrust)
 	}
-	restRespSuccess(w, r, &resp, nil, nil, nil, "Get all sigstore roots of trust")
+
+	resp := make(map[string]interface{})
+	resp["roots_of_trust"] = rootsOfTrust
```

Comment:
The return object's type should be defined in apis.go & described in apis.yaml instead of simply an interface type in the api handler